prover/witness_generator/src/main.rs

```diff
@@ -184,15 +182,23 @@ async fn main() -> anyhow::Result<()> {
                         .await,
                     ),
                 };
-                let kzg_config = KzgConfig::from_env().context("KzgConfig::from_env()")?;
+                let trusted_setup_path =
+                    std::env::var("KZG_TRUSTED_SETUP_PATH").or_else(|err| match err {
+                        std::env::VarError::NotPresent => Ok("./trusted_setup.json".to_owned()),
+                        std::env::VarError::NotUnicode(_) => {
+                            let err = anyhow::Error::new(err)
+                                .context("`KZG_TRUSTED_SETUP_PATH` env variable is invalid");
+                            Err(err)
+                        }
+                    })?;
```

Comment:
I didn't manage to switch to static KZG settings here because they are passed as a file path (rather than `KzgSettings`) inside `zkevm_test_harness` functions.

Reply:
Maybe to unify the approach and simplify the requirements on inputs, we'll do a [lil' hack here](https://docs.rs/tempfile/3.10.1/tempfile/fn.tempfile.html) (obviously wrapped into an `once_cell` for efficiency)? At the same time, we can ask the protocol team to change the public interface in the harness so that `KzgSettings` are passed directly and we can remove it later.